app/models/worksheet.rb

```diff
@@ -5,7 +5,6 @@
 #  id              :bigint           not null, primary key
 #  correct_content :json
 #  display_content :json
-#  name            :string
```

Comment:
sorry, didn't notice this one in the schema
please keep the name  - we need it to differentiate between worksheets

Reply:
Added a name column in the worksheets table 👍🏻